content/tutorials/token-authentication.textile

```diff
@@ -607,7 +607,7 @@ blang[android].
   ...
   dependencies {
       ...
-      compile 'io.ably:ably-android:0.8.7'
+      compile 'io.ably:ably-android:1.0.0'
```

Comment:
Best practice is a matter of debate: http://blog.danlew.net/2015/09/09/dont-use-dynamic-versions-for-your-dependencies/

Last time we discussed it we decided to use a fixed version reference.